tests/init_db.py

```diff
@@ -1,18 +1,33 @@
 # -*- coding: utf-8 -*-
+
 from datetime import date, timedelta
 
 from sqlalchemy import create_engine
 
 from pyramid_oereb.lib import b64
 from pyramid_oereb.lib.adapter import FileAdapter
 from pyramid_oereb.lib.config import Config
+from pyramid_oereb.standard.sources.plr import parse_multiple_standard_themes
 
 file_adapter = FileAdapter()
 
+Config._config = None
+pyramid_oereb_test_yml = 'pyramid_oereb/standard/pyramid_oereb.yml'
+Config.init(pyramid_oereb_test_yml, 'pyramid_oereb')
+
+def create_models():
+    themes = parse_multiple_standard_themes(Config)
+    return themes
```

Comment:
the function is `create_models` but the variable returned is `themes`. This is a bit confusing to read.

Reply:
True, I changed it